src/Cake.Common/Tools/ILRepack/ILRepackSettings.cs

```diff
@@ -0,0 +1,186 @@
+        - /usefullpublickeyforreferences - NOT IMPLEMENTED
+        - /align             - NOT IMPLEMENTED
+        - /closed            - NOT IMPLEMENTED
+        - /allowdup:Type     allows the specified type for being duplicated in input assemblies
+        - /allowduplicateresources allows to duplicate resources in output assembly (by default they're ignored)
+        - /zeropekind        allows assemblies with Zero PeKind (but obviously only IL will get merged)
+        - /wildcards         allows (and resolves) file wildcards (e.g. `*`.dll) in input assemblies
+        - /parallel          use as many CPUs as possible to merge the assemblies
+        - /pause             pause execution once completed (good for debugging)
+        - /verbose           shows more logs
+        - /out:<path>        target assembly path, symbol/config/doc files will be written here as well
+        - <path_to_primary>  primary assembly, gives the name, version to the merged one
+        - <other_assemblies> ...
+        */
+
+        /// <summary>
+        /// Gets or sets the tool path.
+        /// </summary>
+        /// <value>The tool path.</value>
+        public FilePath ToolPath { get; set; }
+
+        /// <summary>
+        /// Gets or sets a keyfile to sign the output assembly
+        /// </summary>
+        /// <value>The keyfile.</value>
+        public string Keyfile { get; set; }
+
+        /// <summary>
+        /// Gets or sets a file to enable logging to (no logging if null or empty)
+        /// </summary>
+        /// <value>The log file.</value>
+        public string Log { get; set; }
+
+        /// <summary>
+        /// Gets or sets the target assembly version.
+        /// </summary>
+        /// <value>The version.</value>
+        public System.Version Version { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether to merge types with identical names into one
+        /// </summary>
+        /// <value><c>true</c> if types with identical names should be merged into one; otherwise, <c>false</c>.</value>
+        public bool Union { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether to disable symbol file generation
+        /// </summary>
+        /// <value><c>true</c> if debug symbols should be disabled; otherwise, <c>false</c>.</value>
+        public bool NDebug { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether to copy assembly attributes (by default only the primary assembly attributes are copied)
+        /// </summary>
+        /// <value><c>true</c> if assembly attributes should be copied; otherwise, <c>false</c>.</value>
+        public bool CopyAttrs { get; set; }
+
+        /// <summary>
+        /// Gets or sets the assembly file to take attributes from
+        /// </summary>
+        /// <value>The assembly file to take attributes from.</value>
+        public string Attr { get; set; }
```

Comment:
Yeah I was debating that but wasn't sure if it wanted just the filename or the full path of the assembly.... Never used this option.